app/scenes/create_channel/create_channel.js

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+import React, {PropTypes, PureComponent} from 'react';
+import {injectIntl, intlShape} from 'react-intl';
+import {
+    Dimensions,
+    Keyboard,
+    Platform,
+    StyleSheet,
+    TouchableOpacity,
+    TouchableWithoutFeedback,
+    View,
+    findNodeHandle
+} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import ErrorText from 'app/components/error_text';
+import FormattedText from 'app/components/formatted_text';
+import TextInputWithLocalizedPlaceholder from 'app/components/text_input_with_localized_placeholder';
+
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import {Constants, RequestStatus} from 'service/constants';
+import EventEmitter from 'service/utils/event_emitter';
+
+import CreateChannelButton from './create_channel_button';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => {
+    const {height, width} = Dimensions.get('window');
```

Comment:
Where it is, this'll only be updated when `theme` changes. I'm 0/5 on where we place this, but it doesn't work like you expect here.

Any way that it could be done by making it flex instead of an explicit height?